modules/rosa-troubleshooting-expired-token.adoc

```diff
@@ -0,0 +1,22 @@
+
+// Module included in the following assemblies:
+//
+// cli_reference/rosa_cli/rosa-troubleshooting-expired-token.adoc
+
+
+[id="rosa-troubleshooting-expired-token_{context}"]
+= Troubleshooting an expired offline access token
+
+If you are using the `rosa` CLI and your api.openshift.com offline access token has expired, an error message appears. This happens when sso.redhat.com invalidates the token.
```

Comment:
```suggestion
If you use the `rosa` CLI and your api.openshift.com offline access token expires, an error message appears. This happens when sso.redhat.com invalidates the token.
```

According the the style guide, it's best to use simple tenses wherever possible. (So, if these edits change something fundamental in the content, please ignore them.)